Rocket.Chat/Controllers/Chat/ChatViewController.swift

```diff
@@ -878,9 +869,26 @@ final class ChatViewController: SLKTextViewController {
         if let previewView = ChatPreviewModeView.instantiateFromNib() {
             previewView.delegate = self
             previewView.subscription = subscription
-            previewView.frame = CGRect(x: 0, y: view.frame.height - previewView.frame.height, width: view.frame.width, height: previewView.frame.height)
+            previewView.translatesAutoresizingMaskIntoConstraints = false
+
             view.addSubview(previewView)
+
+            NSLayoutConstraint.activate([
+                previewView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+                previewView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+                previewView.bottomAnchor.constraint(equalTo: view.bottomAnchor)
+                ])
+
+            collectionView?.bottomAnchor.constraint(equalTo: previewView.topAnchor).isActive = true
```

Comment:
Just for future reference, this constraint is completely valid and does not conflict with any of the constraints added by the `SLKTextViewController`. `SLKTextViewController` has added a height constraint to the `collectionView`, but since it's priority is `750`, the `bottomAnchor` constraint will always take precedence.

This constraint automatically gets removed when the previewView, leaves its superview. This allows for a normal operation when the `textInputbar` is visible and the `SLKTextViewController` takes over the control of the layout.